<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthForm</class>
 <widget class="QWidget" name="AuthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>141</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="login_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="userPassword_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="userName_textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Имя пользователя</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
